### DISCUSSION TITLE
Star Exchange - Let's Support Each Other's GitHub Projects! ⭐️

### DISCUSSION THREAD

**Discussion by 1wos** [Schweinepriester/achievement-testarea#48] [Q&A] [answered]
star exchange ⭐️! If you give a star to my project, [ASF-DataAnalysis](https://github.com/sommizzu/ASF-DataAnalysis), I'll gladly return the favor by starring your project in return ⭐️⭐️⭐️⭐️⭐️⭐️⭐️⭐️⭐️.

**Comment by AdityaNow** [comment, marked as answer]
Done :smiley:  (feel free to star any of my repo https://github.com/AdityaNow/NavtorAssignment & https://github.com/AdityaNow/Wikipedia)
Thank you

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Reply by 1wos** [reply]
I stared your repo ! Please star my repo - 👍🏻 https://github.com/sommizzu/ASF-DataAnalysis

**Reply by 1wos** [reply]
@ternera I stared your repo ! Please star my repo pls ... - 👍🏻 https://github.com/sommizzu/ASF-DataAnalysis

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.

**Comment by KamBRJC** [comment]
https://github.com/z-nnovation/GitSwitcher pls star
